modules/ROOT/pages/_partials/gs-service-discovery-3scale.adoc

```diff
@@ -0,0 +1,26 @@
+[id='gs-service-discovery-3scale']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+= Enabling service discovery in 3scale
+
+By default, a {PRODUCT_SHORT} user cannot perform service discovery in 3scale.
+
+To enable service discovery in 3scale of APIs created in the shared Fuse Online instance:
+
+. Log into OpenShift as the customer-admin user from the command line.
+
+. Enter the following command to enable the `<username>` user perform service discovery of Fuse Online APIs:
```

Comment:
should this be:
> Enter the following command to enable the `<username>` user **to** perform service discovery of Fuse Online APIs

Reply:
@philbrookes  two 'to's is one 'to' too many, maybe
Entering the following command enables the <username> user to perform service discovery of Fuse Online APIs